splitgraph/ingestion/athena/__init__.py

```diff
@@ -0,0 +1,136 @@
+import base64
+import json
+import urllib.parse
+from copy import deepcopy
+from typing import TYPE_CHECKING, Any, Dict, Optional
+
+from splitgraph.core.types import Credentials, Params, TableInfo
+from splitgraph.hooks.data_source.fdw import ForeignDataWrapperDataSource
+from splitgraph.ingestion.common import build_commandline_help
+
+if TYPE_CHECKING:
+    from splitgraph.engine.postgres.engine import PostgresEngine
+
+
+class AmazonAthenaDataSource(ForeignDataWrapperDataSource):
+    credentials_schema: Dict[str, Any] = {
+        "type": "object",
+        "properties": {
+            "aws_access_key_id": {"type": "string", "title": "AWS Access Key Id"},
+            "aws_secret_access_key": {"type": "string", "title": "AWS Secret Access Key"},
+        },
+    }
```

Comment:
These should be required, e.g.:

```suggestion
        "required": ["aws_access_key_id", "aws_secret_access_key"],
    }
```